src/codegen_call_io.cpp

```
@@ -130,498 +130,408 @@ llvm::Value *CodeGen::emitPtrToResult(llvm::Value *ptr, const std::string &name,
-        requireArgs(e, 2);
-        llvm::Value *req = emitExpr(*e.args[0]);
-        llvm::Value *key = emitExpr(*e.args[1]);
-        if (!isHttpRequest(req))
-            codegenError("form_field() requires HttpRequest argument");
-        if (key->getType() != ptrTy_)
-            codegenError("form_field() name must be str");
-        auto fnTy = fnTy_ptr_ptr_to_ptr_;
-        auto fn = mod_->getOrInsertFunction("__ry_http_form_field", fnTy);
-        llvm::Value *result = builder_.CreateCall(fn, {req, key}, "http_ff");
-        return wrapPtrAsOption(result, "http_ff");
+    llvm::Value *portCallback = nullptr;
```

Clang (14.0.6)

[warning] variable 'portCallback' is not initialized

(cppcoreguidelines-init-variables)